models/Asset.php

```diff
@@ -1301,11 +1301,13 @@ public function getCustomSettings(): array
      */
     public function setCustomSettings(mixed $customSettings): static
     {
-        if (is_string($customSettings)) {
+        if (
+            is_string($customSettings) &&
+            !empty($customSettings)
```

Comment:
Variable is always set
```suggestion
            $customSettings !== ''
```